content/applications/inventory_and_mrp/purchase/manage_deals/blanket_orders.rst

```diff
@@ -1,204 +1,176 @@
 
-- :guilabel:`Use lines of agreement`: when creating a new quotation, the product lines will
-  pre-populate with the same products listed on the blanket order, if said blanket order is chosen
-  on the new quotation.
-- :guilabel:`Do not create RfQ lines automatically`: when creating a new quotation and selecting
-  an existing blanket order, the settings will carry over to the new quotation, but the product
-  lines will not populate.
+- :guilabel:`Use lines of agreement`: when creating a new quotation, the product lines pre-populate
+  with the same products listed on the blanket order, if said blanket order is chosen for the new
+  quotation.
+- :guilabel:`Do not create RfQ lines automatically`: when creating a new quotation, **and**
+  selecting an existing blanket order, the settings will carry over to the new quotation, but the
+  product lines will **not** populate.
 
 And, there are two options that can be activated for :guilabel:`Quantities`:
 
 - :guilabel:`Use quantities of agreement`: when creating a new quotation, the product quantities
-  listed on the blanket order will pre-populate on the product lines, if said blanket order is
-  chosen on the new quotation.
-- :guilabel:`Set quantities manually`: when creating a new quotation and selecting an existing
-  blanket order, the product lines will pre-populate, but all quantities will be set to **0**. The
-  quantities will need to be manually set by the user.
-
-Once the desired changes have been made (if any), click :guilabel:`New` (via the breadcrumbs, at
-the top of the page) to navigate back to the blanket order form, and click :guilabel:`Confirm` to
-save this new purchase agreement. Once confirmed, the blanket order changes from *Draft* to
-*Ongoing*, meaning this agreement can be selected and used when creating new :abbr:`RFQs (Requests
-for Quotation)`.
+  listed on the blanket order pre-populate on the product lines, if said blanket order is chosen for
+  the new quotation.
+- :guilabel:`Set quantities manually`: when creating a new quotation, **and** selecting an existing
+  blanket order, the product lines pre-populate, but all quantities will be set to **0**. The
+  quantities **must** be manually set by the user.
+
+Once any desired changes have been made, click :guilabel:`New` (via the breadcrumbs, at the top of
+the page) to navigate back to the blanket order form. Then, click :guilabel:`Confirm` to save this
+new purchase agreement. Once confirmed, the blanket order changes from *Draft* to *Ongoing*, meaning
+this agreement can be selected and used when creating new |RfQs|.
 
 .. tip::
    After creating and confirming a blanket order, products, quantities, and prices can still be
    edited, added, and removed from the purchase agreement.
 
-Create a new :abbr:`RFQ (Request for Quotation)` from the blanket order
-=======================================================================
+Create a new |RfQ| from the blanket order
```

Comment:
oh wow I didn't know we could put abbreviations into the headings like this! Cool find, Brandon